src/radix_connect/mobile/relay_service/models/request/request.rs

```diff
@@ -0,0 +1,306 @@
+use super::request_method::Method;
+use crate::prelude::*;
+use crate::radix_connect::mobile::session::session_id::SessionID;
+
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq)]
+#[serde(rename_all = "camelCase")]
+pub struct Request {
+    pub method: Method,
+    pub session_id: SessionID,
+    pub data: Option<BagOfBytes>,
```

Comment:
do we plan to UniFFI export this? If not, prefer using `Vec<u8>` - we very much hope to remove `BagOfBytes` some days. The intended usage is to use `BagOfBytes` in all UniFFI APIs (functions) and only in UniFFI exported **types**.

Reply:
no, this is not meant to be expose through UniFFI